save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/HttpUtils.kt

```diff
@@ -43,31 +49,29 @@ fun ByteReadChannel.toByteBufferFlux(): Flux<ByteBuffer> = DataBufferUtils.readI
  *
  * @param urlPathSegments list of endpoint path segments (e.g. ["run"])
  * @param demoPod [Pod] with save-demo-agent that runs requested demo
- * @param kc [KubernetesClient] that is required for kubernetes interactions
- * @param request callback that receives url and port
+ * @param request callback that receives [Url]
  * @return result of type [R]
  * @throws IOException rethrown from [request]
  */
-suspend fun <R> demoAgentRequestWrapper(
+private suspend fun <T> KubernetesClient.portForwardingRequest(
```

Comment:
do we still need `portForwardingRequest`?

Reply:
I would like to leave it here for now just in case. Do now want my time to be spent so useless